frontend/packages/shared/src/components/molecules/AltinnMenuItem.tsx

```diff
@@ -4,7 +4,7 @@ import classes from './AltinnMenuItem.module.css';
 
 export interface IAltinnMenuItemProps {
   text: string;
-  iconClass: string;
+  iconClass?: React.ComponentType;
```

Comment:
Denne bør nok ha et annet navn siden det ikke lengre er en klasse. Hva med bare `icon`? Trenger den for øvrig å være valgfri? Den var jo ikke det tidligere.

Reply:
fikset

Reply:
Jeg tror ikke spørsmålstegnet er nødvendig heller, siden ikonet var påkrevd fra før.

Reply:
den har ikke spørsmålstegnet , jeg fjernet den med første kommentar  🤔

Reply:
Hmm, da er det bare Github som har tullet litt og vist meg feil versjon 😅